tests/cases/fourslash/codeFixImportNonExportedMember.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference path='fourslash.ts' />
+// @Filename: /a.ts
+////declare function zoo(): any;
+////export { zoo };
+
+// @Filename: /b.ts
+////declare function foo(): any;
+////declare function bar(): any;
+////export { foo };
+
+// @Filename: /c.ts
+////import { zoo } from "./a";
+////import { bar } from "./b";
+
+goTo.file("/c.ts");
+verify.codeFixAvailable([
+    { description: `Export 'bar' from module './b'` },
+    { description: `Remove import from './a'` },
+    { description: `Remove import from './b'` },
+]);
+verify.codeFix({
+    index: 0,
+    description: `Export 'bar' from module './b'`,
+    newFileContent: {
+        '/b.ts': `declare function foo(): any;
+declare function bar(): any;
+export { foo };
+
+export { bar };

```

Comment:
I don't think this is the desired output. Here's a few cases to keep in mind:

## The module only uses `export { ... }` to export

If there is an `export { ... }` statement with no module specifier (i.e. what you have above), and no other declaration in the module has an `export` keyword in front of it, then you want to add an identifier to the first `export { ... }` with no module specifier.

## The declaration is part of a variable statement with multiple variables

```ts
// @filename: a.ts
let a = 1, b = 2, c = 3;
export function whatever() {
}

// @filename: b.ts
import { a } from "./a"
```

In this case, you don't necessarily want to split the variables up. It makes sense to find an existing `export { ... }` statement, or generate a new one.

## Other cases

In any other case, I think that you just want to add an `export` modifier to the declaration.

Reply:
Thanks for your advice, I will follow the principles below.

- if the locals originate from a `.d.ts` file, do nothing.
- No export found in the file: 
       a.The declaration is part of a variable statement with multiple variables, creating `export {...}`
       b.The declaration is a single function or variable: add `export`
- more than one `export {...}` found in the file:
        a.The declaration is part of a variable statement with multiple variables or The declaration is a single variable, add the declaration to the first `export {...}`
        b.The declaration is a single function: add `export`

in order to finish the above task, I need three key messages.

- Get all of the export declaration from a SourceFile.
- judge whether the declaration is a part of a variable.
- judge whether the declaration is a function or variable.

@DanielRosenwasser best wishes if you can tell me how to get this messages.